static/js/settings_users.js

```diff
@@ -428,19 +428,44 @@ function get_human_profile_data(fields_user_pills) {
 }
 
 export function confirm_deactivation(user_id, handle_confirm, loading_spinner) {
+    const bots_owned_by_user = bot_data.get_all_bots_owned_by_user(user_id);
     const user = people.get_by_user_id(user_id);
     const opts = {
         username: user.full_name,
         email: settings_data.email_for_user_settings(user),
+        bots_owned_by_user,
     };
     const html_body = render_settings_deactivation_user_modal(opts);
 
+    function fill_number_of_invites_by_user() {
+        channel.get({
+            url: "/json/invites",
+            idempotent: true,
+            timeout: 10 * 1000,
+            success(data) {
+                let number_of_invites_by_user = 0;
+                for (const invite of data.invites) {
+                    if (invite.invited_by_user_id === user_id) {
+                        number_of_invites_by_user = number_of_invites_by_user + 1;
+                    }
+                }
+                $("#deactivate-user-modal .number_of_invites_by_user").text(
+                    number_of_invites_by_user,
+                );
+            },
+            // We display 0 number of invites as the default. Knowing the number of invites
+            // requires making this request. If it fails, we do nothing.
+        });
+    }
```

Comment:
So, this approach is going to result in a brief flash where `0` is displayed regardless of the true value, which seems bad.

What we should do instead is call `confirm_dialog.launch` from the success handler, so that we have all the data when we render the deactivation modal -- and then we can just fill in this value in template rendering, not via JavaScript.